runtime/src/lib.rs

```diff
@@ -431,6 +404,7 @@ impl pallet_treasury::Config for Runtime {
     // type Tippers = GeneralCouncilProvider;
     // Just gets burned.
     type WeightInfo = ();
+    type MaxApprovals = MaxApprovals;
 }
 
 parameter_types! {
```

Comment:
why don't we need pallet_staking in the parachain?

Reply:
are there any other runtime modules that are in the standalone chain that aren't currently in the parachain and should be ?

Reply:
`pallet_staking` is commented out https://github.com/DataHighway-DHX/node/blob/ilya/parachain-update/runtime/src/lib.rs#L480 in existing parachain. I don't know why. May be @festelo can answer

Regarding Identify and implement runtime modules that are in the standalone chain that aren't currently in the parachain, I have created an issue.
https://github.com/DataHighway-DHX/node/issues/227